crates/uv-cli/src/lib.rs

```diff
@@ -232,7 +232,7 @@ pub struct GlobalArgs {
     /// Hide all progress outputs.
     ///
     /// For example, spinners or progress bars.
-    #[arg(global = true, long)]
+    #[arg(global = true, long, env = EnvVars::UV_NO_PROGRESS, value_parser = clap::builder::BoolishValueParser::new(), overrides_with("no_progress"))]
```

Comment:
Is there a reason it overrides with `no_progress`? Do you mean `"progress"` there?

Reply:
Good point. I did a copy paste of the `preview` field and didn't think very hard. I removed the `overrides_with` parameter because there isn't a `progress` field.